relayer/chains/evm/provider.go

```diff
@@ -496,13 +495,8 @@ func (p *Config) GetConnContract() string {
 	return p.Contracts[providerTypes.ConnectionContract]
 }
 
-func (p *Provider) SignMessage(message []byte) ([]byte, error) {
-	hash := sha3.NewLegacyKeccak256()
-	hash.Write([]byte("\x19Ethereum Signed Message:\n" + fmt.Sprint(len(message)) + string(message)))
-	messageHash := hash.Sum(nil)
-	wallet, err := p.Wallet()
-	if err != nil {
-		return nil, err
-	}
-	return crypto.Sign(messageHash, wallet.PrivateKey)
+func (p *Provider) GetSignMessage(message []byte) []byte {
```

Comment:
I don't think we need this in Provider. Because we have everything we need the relayer core itself.

Reply:
This gives chain ability to define and hashing algorithm to choose.